assets/js/custom/UserNotifications.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-env jquery */
+/* global Swal */
+/* global FetchNotifications */
+
+// eslint-disable-next-line no-unused-vars
+function UserNotifications (markAsReadUrl, countNotificationsUrl, somethingWentWrongError,
+  notificationsClearError) {
+  const self = this
+  let all = true
+  let follower = false
+  let comment = false
+  let reactions = false
+  let remixes = false
+  self.markAsReadUrl = markAsReadUrl
+  self.countNotificationsUrl = countNotificationsUrl
+  self.somethingWentWrongError = somethingWentWrongError
+  self.notificationClearError = notificationsClearError
+
+  $(document).on('click', '#all-notif', function () {
+    if (all === false) {
+      $(this).css('background-color', '#acacac')
```

Comment:
define a function to set the background and color of an element (highly reduces the duplicated code)